### ISSUE TITLE
Usability issue: selecting semaphore in the dashboard should disable the options related to thread and vice versa

### ISSUE THREAD

**Issue by sriniprash** [yammer/breakerbox#38]
@chrisgray 

I have found a usability issue in the breakerbox dashboard:

Selecting semaphore in the dashboard should disable the options related to thread and vice versa.

Is this the expected use case? This can be confusing. I can fix this in case this is not expected.

**Comment by chrisgray** [comment]
I agree this would be a nice addition! As well as enforcing the user to select one of the possible execution isolation strategy options.
